test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ActionSelectorTest.cs

```diff
@@ -790,7 +791,7 @@ private ControllerActionDescriptorProvider GetActionDescriptorProvider()
 
             var manager = GetApplicationManager(controllerTypes);
 
-            var modelProvider = new DefaultApplicationModelProvider(options);
+            var modelProvider = new DefaultApplicationModelProvider(options, TestModelMetadataProvider.CreateDefaultProvider());
```

Comment:
Still using `TestModelMetadataProvider.CreateDefaultProvider()` and `TestModelMetadataProvider.CreateDefaultProvider(IList<IMetadataDetailsProvider>)` a great deal _outside_ integration tests. Think the count of new calls in unit tests is almost 25.

This is a concern because it brings so many providers into our _unit_ tests. Can this be avoided or reduced with something like a new `TestModelMetadataProvider` constructor accepting a `bool configureAdditionalModelMetadataDetailsProviders` parameter?